### ISSUE TITLE
Line break/new line is created when adding 2+ spaces to the end of a line

### ISSUE THREAD

**Issue by dvaerum** [charmbracelet/glow#145]
It doesn't look like glow is following the standard for markdown when it comes to the option of adding 2+ spaces to the end of a line to indicated line break/new line.

>When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return.

**Ref:** https://daringfireball.net/projects/markdown/syntax#p

**Note:** I am using (the latest) version 0.2.0 of glow

**Comment by bionade24** [comment]
I experience the same issue and can confirm it.

**Comment by muesli** [comment]
You're right, we're currently re-flowing the text to fit the desired output width. We should make this optional.
